src/bindings/intercept/env/src/getenv.cpp

```diff
@@ -548,7 +548,7 @@ extern "C" pid_t fork ()
 
 Key * elektraContextEvaluation (ELEKTRA_UNUSED KeySet * ks, ELEKTRA_UNUSED Key * key, Key * found, option_t option)
 {
-	if (found && !strncmp (keyName (found), "spec/", 5) && option == KDB_O_CALLBACK)
```

Comment:
@markus2330 what is the reason for checking the `KDB_O_CALLBACK` flag so strict? With such a strict checking it is impossible to set at the most outer `ksLookup(...)` invocation the `KDB_O_OPMPHM` flag.

Reply:
The `KDB_O_CALLBACK ` interface is experimental, so change proposals are very welcomed. Adding more then one callback would also be interesting.  #867 proposes to allow plugins for ksLookup (using the plugins and not `KDB_O_CALLBACK ` as interface).